### ISSUE TITLE
Display all topics/articles when no argument is passed

### ISSUE THREAD

**Issue by Nowaker** [filiparag/wikiman#11]
Current behavior:
```
% wikiman                      
find: ‘/home/nowaker/.config/wikiman/sources’: No such file or directory
error: empty search query
```

**Comment by filiparag** [comment]
This feature was present in earlier version, but due to performance issues, it was removed in commit [a801157ca94143128967a9fcc8f1efb7168f9ad8](https://github.com/filiparag/wikiman/tree/a801157ca94143128967a9fcc8f1efb7168f9ad8).

In the meantime, source module structure was improved, so adding it back shouldn't be a problem.

**Comment by filiparag** [comment]
List all pages feature is added back in [e9d08fb](https://github.com/filiparag/wikiman/commit/e9d08fb63c7547343258cc897d62f0592de0826e).
Error when sources are missing was fixed in [3c9ae3b](https://github.com/filiparag/wikiman/commit/3c9ae3b1cb7f3428e1a3b3027114860a9ec2eff7)
